tools/knb/README.md

```diff
@@ -0,0 +1,112 @@
+# knb - kn builder
+
+CLI tool to enhance plugin build experience for [Knative Client](https://github.com/knative/client).
+
+
+### Build
+```bash
+go build
+```
+
+### Install
+```bash
+go get knative.dev/client/tools/knb
+```
+
+### Usage
+
+##### Create custom `kn` distribution
+
+The `knb` can be used to generate enhanced `kn` source files with inlined plugins.
+
+Create configuration file `.kn.yaml` in a root directory of `knative/client` that should specify at least `name, module, version` coordinates of the plugin.
+The `plugin distro` will generated the required go files and add dependency to `go.mod`.
```

Comment:
Not clear what the `plugin distro` is?

Reply:
I know it’s a command of the `knb` tool but not clear here

Reply:
Fixed in https://github.com/knative/client/pull/1226/commits/a06c275eb3d0ae388efb1441679cb36444c5f741.